sdk/documenttranslation/azure-ai-documenttranslation/tests/testcase.py

```diff
@@ -113,6 +120,148 @@ def generate_sas_url(self, container_name, permission):
         container_sas_url = self.storage_endpoint + container_name + "?" + sas_token
         return container_sas_url
 
-    def wait(self):
+    def wait(self, duration=30):
         if self.is_live:
-            time.sleep(30)
+            time.sleep(duration)
+
+
+    # model helpers
+    def _validate_doc_status(self, doc_details, target_language):
+        # specific assertions
+        self.assertEqual(doc_details.status, "Succeeded")
+        self.assertIsNotNone(doc_details.translate_to, target_language)
+        # generic assertions
+        self.assertIsNotNone(doc_details.id)
+        self.assertIsNotNone(doc_details.translated_document_url)
+        self.assertIsNotNone(doc_details.translation_progress)
+        self.assertIsNotNone(doc_details.characters_charged)
+        self.assertIsNotNone(doc_details.created_on)
+        self.assertIsNotNone(doc_details.last_updated_on)
+
+    def _validate_translation_job(self, job_details, **kwargs):
+        status = kwargs.pop("status", None)
+        total = kwargs.pop('total', None)
+        failed = kwargs.pop('failed', None)
+        succeeded = kwargs.pop('succeeded', None)
+        inprogress = kwargs.pop('inprogress', None)
+        notstarted = kwargs.pop('notstarted', None)
+        cancelled = kwargs.pop('cancelled', None)
+        # status
+        self.assertEqual(job_details.status, status) if status else self.assertIsNotNone(job_details.status)
+        # docs count
+        self.assertEqual(job_details.documents_total_count, total) if total else self.assertIsNotNone(job_details.documents_total_count)
+        self.assertEqual(job_details.documents_failed_count, failed) if failed else self.assertIsNotNone(job_details.documents_failed_count)
+        self.assertEqual(job_details.documents_succeeded_count, succeeded) if succeeded else self.assertIsNotNone(job_details.documents_succeeded_count)
+        self.assertEqual(job_details.documents_in_progress_count, inprogress) if inprogress else self.assertIsNotNone(job_details.documents_in_progress_count)
+        self.assertEqual(job_details.documents_not_yet_started_count, notstarted) if notstarted else self.assertIsNotNone(job_details.documents_in_progress_count)
+        self.assertEqual(job_details.documents_cancelled_count, cancelled) if cancelled else self.assertIsNotNone(job_details.documents_cancelled_count)
+        # generic assertions
+        self.assertIsNotNone(job_details.id)
+        self.assertIsNotNone(job_details.created_on)
+        self.assertIsNotNone(job_details.last_updated_on)
+        self.assertIsNotNone(job_details.total_characters_charged)
+
+    def _validate_format(self, format):
+        #self.assertIsNotNone(format.format)
```

Comment:
Ah I see the problem. I had to rename this property from `format` to `file_format` in this [PR ](https://github.com/Azure/azure-sdk-for-python/pull/17552/files) as to not re-define a built-in name in Python. When you pulled the latest master I think you got my changes. If you change to `format.file_format` does it pass the tests?